apps/oxlint/src/snapshots/fixtures__extends_config_overrides@oxlint.snap

```diff
@@ -31,7 +30,16 @@ working directory: fixtures/extends_config
    `----
   help: Avoid using ambiguous text like "click here", replace it with more descriptive text that provides context.
 
-Found 0 warnings and 3 errors.
+  ! ]8;;https://oxc.rs/docs/guide/usage/linter/rules/jsx_a11y/anchor-is-valid.html\eslint-plugin-jsx-a11y(anchor-is-valid)]8;;\: Missing `href` attribute for the `a` element.
```

Comment:
this is a correctness rule, that I think should've been enabled before, but wasn't